ui/apps/platform/cypress/integration/accessControl/accessControl.helpers.js

```diff
@@ -198,8 +196,6 @@ export function visitAccessControlEntitiesWithStaticResponseForPermissions(
 ) {
     const entitiesPath = getEntitiesPath(entitiesKey);
     visitWithStaticResponseForPermissions(entitiesPath, staticResponseForPermissions);
-
-    cy.get(`h1:contains("${containerTitle}")`);
```

Comment:
It's not clear to me why you are removing this assertion?

Reply:
Good eyes.

This function visits with permissions, so `h1` element might have **Cannot find the page**.